weave/trace_server/migrations/003_feedback.up.sql

```diff
@@ -0,0 +1,52 @@
+
+CREATE TABLE feedback (
+    /*
+    `id`: The unique identifier for the feedback. This is a UUID.
+    */
+    id String,
+
+    /*
+    `project_id`: The project identifier for the ref. This is an internal
+    identifier that matches the project identifier in the W&B API.
+    It is stored for feedback to allow efficient permissions filtering.
+    */
+    project_id String,
+
+    /*
+    `wb_ref`: The ref the feedback is associated with.
+    Note: the wb prefix is to avoid conflict with React's notion of ref.
+    */
+    wb_ref String,
+
+    /*
+    `wb_user_id`: The ID of the user account used to authenticate the feedback creation.
+    This is the ID of the user in the W&B API.
+    */
+    wb_user_id String,
+
+    /*
+    `creator`: The name to display for who the feedback came from. Can default to the name of
+    the user account used to authenticate the feedback creation, but can be an arbitrary string.
+    This is useful for feedback that originated with end users who may not have a W&B account.
+    */
+    creator String,
```

Comment:
if this happens to be the W&B username (common case?) then we might get out of sync in the event of a deletion / change. Perhaps this should be nullable in which case the null value indicates we should use the wb_user_id and lookup the name at display time?